README.md

```diff
@@ -39,6 +39,7 @@ $ pytest --cov=.
 | `DJ05` | URLs include() should set a namespace |
 | `DJ06` | ModelForm should not set exclude, instead it should use fields, which is an explicit list of all the fields that should be included in the form |
 | `DJ07` | ModelForm.Meta should not set fields to '__all__'|
+| `DJ08` | Models that inherits from django db models should set '__str__'|
```

Comment:
nit pick: could you put str inside ` ` so that it is highlighted? (I'm just seeing that the '__all__' on top is wrong too, I will fix it)

Something like:

| `DJ08` | Models that inherits from django db models should set `__str__`|

Instead of:

| `DJ08` | Models that inherits from django db models should set '__str__'|

Reply:
I can fix both.

Reply:
I forgot to tell you that I made the requested changes.